enterprise/course_catalog_api.py

```diff
@@ -29,6 +29,8 @@ class CourseCatalogApiClient(object):
     Object builds an API client to make calls to the Catalog API.
     """
 
+    DEFAULT_VALUE_SAFEGUARD = "DEFAULT_VALUE_SAFEGUARD"
```

Comment:
Nit: In edx-platform, I've seen `object()` used for sentinel values ([example1](https://github.com/edx/edx-platform/blob/62d129fdab1e047299441b78031a030308460464/common/lib/capa/capa/inputtypes.py#L130-L132), [example2](https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/lms/djangoapps/courseware/masquerade.py#L186)). It doesn't really make any practical difference, but is a bit cleaner form the theoretical perspective.